src/fe/inf_fe_map.C

```diff
@@ -29,6 +29,18 @@
 namespace libMesh
 {
 
+bool libmesh_isfinite(Real value)
+{
+  // check if \p value is infinite
+  if( value > std::numeric_limits<Real>::max()
+         || value <  std::numeric_limits<Real>::min() )
```

Comment:
ah! of course.
But this 
'''if( std::abs(value) > std::numeric_limits<Real>::max())'''
should work, doesn't it?